code/modules/paperwork/paperplane.dm

```diff
@@ -0,0 +1,142 @@
+// Ported from TG
+/obj/item/weapon/paperplane
+	name = "paper plane"
+	desc = "Paper, folded in the shape of a plane."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "paperplane"
+	throw_range = 7
+	throw_speed = 1
+	throwforce = 0
+	w_class = WEIGHT_CLASS_TINY
+	burn_state = FLAMMABLE
+	burntime = 5
+
+	var/obj/item/weapon/paper/internal_paper
+
+/obj/item/weapon/paperplane/New(loc, obj/item/weapon/paper/new_paper)
+	. = ..()
+	pixel_y = rand(-8, 8)
+	pixel_x = rand(-9, 9)
+	if(new_paper)
+		internal_paper = new_paper
+		flags = new_paper.flags
+		color = new_paper.color
+		new_paper.forceMove(src)
+	else
+		internal_paper = new /obj/item/weapon/paper(src)
+	update_icon()
+
+/obj/item/weapon/paperplane/Destroy()
+	if(internal_paper)
+		qdel(internal_paper)
+		internal_paper = null
+	return ..()
+
+/obj/item/weapon/paperplane/suicide_act(mob/living/user)
+	user.Stun(10)
+	user.visible_message("<span class='suicide'>[user] jams [src] in \his nose. It looks like \he's trying to commit suicide!</span>")
```

Comment:
This might use the paper plane's gender instead of the users

Reply:
I sexually Identify as a paper airplane. Ever since I was a boy I dreamed of soaring over classrooms nailing shitfaced students in the head. People say to me that a person being a folded sheet of paper is Impossible and I’m fucking retarded but I don’t care, I’m beautiful. I’m having a plastic surgeon give me glass bones and paper skin, and then skillfully fold me into the perfect form for gliding through the air. From now on I want you guys to respect my right to glide through the air freely. If you can’t accept me you’re a paperphobe and need to check your origami privilege. Thank you for being so understanding.

Someone stab me